src/Pages/cart.css

```diff
@@ -0,0 +1,20 @@
+.cart-wrapper {
+  border: 3px solid black;
+  max-width: 80%;
+  margin: 0 auto;
+}
+
+.cart-grid {
+  border: 3px solid red;
+  display: grid;
+  gap: 1rem;
+  grid-template-columns: 3fr 2fr;
+}
+
+.cart-products-container {
+  border: 3px solid yellow;
+}
+
+.cart-price-container {
+  border: 3px solid purple;
```

Comment:
have seen puple color used twice so you can use variable for that

Reply:
this was just a skeleton for the project so I focused more on the functionality rather than thinking about the design and the CSS part. I will make a branch and design the whole the thing up. due to time constraints I first focused on the functionaity.